### PR TITLE
Implement getting and creating block producers comments component

### PR DESCRIPTION
**Pull request by anastasiia-bilova**

### CODE REVIEW

File: src/components/pages/BlockProducerPage.vue [hunk 1]
```diff
@@ -41,13 +70,29 @@ export default {
         user: null,
         id: null,
       },
+      comments: null,
+      commentsCount: null,
```
Comment (dmytrostriletskyi) [inline src/components/pages/BlockProducerPage.vue]: `commentsCount` > `commentsNumber`.

File: src/store/modules/comment.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    getComments (state, comments) {
+      state.comments = comments
+      state.commentsCount = comments.length
+    },
+    createComment (state, isCreated) {
+      state.isCreated = isCreated
+    },
+  },
+  actions: {
+    get({ commit }, { identifier }) {
```
Comment (dmytrostriletskyi) [inline src/store/modules/comment.js]: `identifier` > `blockProducerIdentifier`

**Review by dmytrostriletskyi — APPROVED** [review]

File: src/components/pages/BlockProducerPage.vue (hunk 1 above)
Comment (anastasiia-bilova) [reply src/components/pages/BlockProducerPage.vue]: @dmytrostriletskyi, done.

File: src/store/modules/comment.js (hunk 2 above)
Comment (anastasiia-bilova) [reply src/store/modules/comment.js]: @dmytrostriletskyi, done.

**Pull request merged**